projects/wvr-elements/src/lib/wvr-editor/wvr-editor.component.ts

```diff
@@ -0,0 +1,92 @@
+import { AfterViewInit, Component, EventEmitter, Injector, Input, Output } from '@angular/core';
+import tinymce from 'tinymce';
+import { WvrBaseComponent } from '../shared/wvr-base.component';
+
+@Component({
+  selector: 'wvr-editor-component',
+  templateUrl: './wvr-editor.component.html',
+  styleUrls: ['./wvr-editor.component.scss']
+})
+export class WvrEditorComponent extends WvrBaseComponent implements AfterViewInit {
+
+  htmlId = `wvr-editor-${this.id}`;
+
+  @Input() apiKey = 'no-api-key';
+
+  @Input() cloudChannel = '5-stable';
+
+  @Input() skinUrl =  `${this.appConfig.assetsUrl}/skins/ui/oxide-dark`;
+
+  @Input() initialValue = '';
+
+  @Input() disabled: 'true' | 'false' = 'false';
+
+  @Input() menuBar = 'file edit view insert format table tools help';
```

Comment:
Lets use json configuration for this.

Reply:
Utilizing json format for menu instead of Object